opencog/nlp/relex2logic/rules/tense.scm

```diff
@@ -0,0 +1,64 @@
+;Both link grammar and relex output are selected because of the pattern 
+;Example : "He gave me the book."
+;(DefinedLinguisticConceptNode "past") and (DefinedLinguisticConceptNode ".v-d")
+;I just want to avoid the Link Grammar output. There might be another way to do this
+(define (check-tense tense)
+    (if (string-contains (cog-name tense) ".")
```

Comment:
right. The would need to be added. Adding them is not hard, you edit nlp-types.script and add them.